bin/Powheg/run_pwg.py

```diff
@@ -338,7 +338,7 @@ def runGetSource(parstage, xgrid, folderName, powInputName, process, tagName) :
 fi
 
 ### retrieve the powheg source tar ball
-export POWHEGSRC=powhegboxV2rwgt_Mar2017.tar.gz
+export POWHEGSRC=powhegboxV2_rev3407_date20170711.tar.gz
 
```

Comment:
rwgt is in fact removed. 
The new naming is given automatically by the create_source script (of which I am also not an author)